Examples/ArcGISToolkitExamples/VCListViewController.swift

```diff
@@ -15,12 +15,11 @@ import UIKit
 import ArcGISToolkit
 
 open class VCListViewController: UITableViewController {
-    
     public var storyboardName: String?
     
     public var viewControllerInfos: [(vcName: String, viewControllerType: UIViewController.Type, nibName: String?)] = [
-        ]{
-        didSet{
+        ] {
```

Comment:
What would you think of moving this up to the previous line? It looks a little odd here on its own.